test/FlashCsrfGuardTest.php

```diff
@@ -11,10 +11,9 @@
 
 class FlashCsrfGuardTest extends TestCase
 {
-    /** @var MockObject<FlashMessagesInterface>  */
-    private $flash;
-    /** @var FlashCsrfGuard */
-    private $guard;
+    /** @var MockObject&FlashMessagesInterface  */
+    private MockObject $flash;
```

Comment:
Same here? More of a question than anything else - I always make the type hint the collaborator/dependency and document the MockObject

Reply:
updated https://github.com/mezzio/mezzio-csrf/pull/15/commits/049102261d02070f5eed20985ce2e862da9aa811